# local-repo translation file
# Copyright (C) 2012 ushi
# This file is distributed under the same license as the local-repo package.
# 
# Translators:
# Hector Mtz-Seara <hseara@gmail.com>, 2012.
msgid ""
msgstr ""
"Project-Id-Version: local-repo\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2012-05-26 09:07+0200\n"
"PO-Revision-Date: 2012-05-26 08:21+0000\n"
"Last-Translator: martinkalcher <martin.kalcher@googlemail.com>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: ca\n"
"Plural-Forms: nplurals=2; plural=(n != 1)\n"

#: ../local-repo:30
msgid ""
"This program helps to manage local repositories. Specify the path to the\n"
"repository with the first argument. If no option is specified, some repo\n"
"information will be printed."
msgstr ""

#: ../local-repo:33
#, python-format
msgid "%(prog)s path [options]"
msgstr "%(prog)s ruta [opcions]"

#: ../local-repo:39 ../local-repo:42 ../local-repo:65
msgid "path"
msgstr "ruta"

#: ../local-repo:40
msgid "path to the repo or the repo name if set up in your config file '{0}'"
msgstr ""

#: ../local-repo:43
msgid ""
"add a package to the repo - path can point to a local or remote package "
"file, pkgbuild file or pkgbuild tarball - supported protocols are HTTP(S) "
"and FTP"
msgstr ""

#: ../local-repo:46 ../local-repo:49 ../local-repo:68 ../local-repo:74
msgid "name"
msgstr "nom"

#: ../local-repo:47
msgid "download, build and add a package from the AUR to the repo"
msgstr "Baixa, compila i afegeix un paquet de AUR al repositori."

#: ../local-repo:50
msgid ""
"rebuild specified packages - NOTE: the 'pkgbuild' path must be set up in the"
" config file"
msgstr ""

#: ../local-repo:54
msgid "run an integrity check"
msgstr "Executa una comprovació d'integritat"

#: ../local-repo:57
msgid "clear the cache"
msgstr ""

#: ../local-repo:60
msgid "the elephant never forgets"
msgstr "L'elefant mai oblida "

#: ../local-repo:63
msgid ""
"force an operation - use this with -a or --add to up-/downgrade a package"
msgstr ""

#: ../local-repo:66
msgid "use an alternative config file (instead of '{0}')"
msgstr ""

#: ../local-repo:69
msgid "display info for specified packages"
msgstr "mostrar la informació per als paquets especificats"

#: ../local-repo:72
msgid "list all packages from the repo"
msgstr "llista de tots els paquets del repositori"

#: ../local-repo:75
msgid "remove packages from the repo"
msgstr "esborra els paquets del repositori"

#: ../local-repo:78
msgid "restore repo database"
msgstr ""

#: ../local-repo:80
msgid "term"
msgstr "terme"

#: ../local-repo:81
msgid "find packages"
msgstr "troba paquets"

#: ../local-repo:84
msgid "upgrade all packages in the repo, which are available in the AUR"
msgstr "actualitza tots els paquets al repositori que estiguin disponibles en AUR"

#: ../local-repo:87
msgid ""
"upgrade all packages in the repo, which are based on a VCS and available in "
"the AUR"
msgstr ""

#: ../localrepo/aur.py:104
msgid "Could not reach the AUR"
msgstr "No s'ha pogut connectar amb AUR "

#: ../localrepo/aur.py:107 ../localrepo/aur.py:117
msgid "AUR responded with error: {0}"
msgstr "AUR ha respòs amb l'error:{0} "

#: ../localrepo/aur.py:114 ../localrepo/aur.py:125
msgid "AUR responded with invalid data"
msgstr "AUR ha respòs amb dades no vàlides"

#: ../localrepo/config.py:55
msgid "Could not open config file: {0}"
msgstr ""

#: ../localrepo/config.py:60
msgid "Could not parse config file: {0}"
msgstr ""

#: ../localrepo/config.py:151
msgid "Could not save config file: {0}"
msgstr ""

#: ../localrepo/localrepo.py:35
msgid "Execution cancelled by user"
msgstr "Execució cancel·lada per l'usuari "

#: ../localrepo/localrepo.py:53
msgid "Loading repo: {0}"
msgstr ""

#: ../localrepo/localrepo.py:69
msgid "This repo has no packages"
msgstr "Aquest repositori no conté paquets"

#: ../localrepo/localrepo.py:80
msgid "Package does not exist: {0}"
msgstr ""

#: ../localrepo/localrepo.py:83
msgid "Package information: {0}"
msgstr ""

#: ../localrepo/localrepo.py:92
msgid "No package found"
msgstr "Paquet no trobat"

#: ../localrepo/localrepo.py:101
msgid ""
"Need following packages as dependencies:\n"
"[{0}]"
msgstr ""

#: ../localrepo/localrepo.py:103
msgid "Install?"
msgstr ""

#: ../localrepo/localrepo.py:104
msgid "Try without installing dependencies?"
msgstr ""

#: ../localrepo/localrepo.py:107 ../localrepo/localrepo.py:238
#: ../localrepo/localrepo.py:267
msgid "Bye"
msgstr "Adeu"

#: ../localrepo/localrepo.py:119
msgid ""
"Installed following packages as dependencies:\n"
"[{0}]"
msgstr ""

#: ../localrepo/localrepo.py:121
msgid "Uninstall?"
msgstr ""

#: ../localrepo/localrepo.py:130 ../localrepo/localrepo.py:131
msgid "Forging a new package: {0}"
msgstr ""

#: ../localrepo/localrepo.py:158
msgid "Adding package to the repo: {0}"
msgstr ""

#: ../localrepo/localrepo.py:160
msgid "Added Package: {0} {1}"
msgstr ""

#: ../localrepo/localrepo.py:168
msgid "Please specify 'pkgbuild' in your config file!"
msgstr ""

#: ../localrepo/localrepo.py:178
msgid "Packages do not exist: {0}"
msgstr ""

#: ../localrepo/localrepo.py:181
msgid "Removing packages: {0}"
msgstr ""

#: ../localrepo/localrepo.py:185
msgid "Removed packages: {0}"
msgstr ""

#: ../localrepo/localrepo.py:192 ../localrepo/localrepo.py:216
#: ../localrepo/localrepo.py:254
msgid "Retrieving package info from the AUR"
msgstr "Rebent informació del paquet d'AUR"

#: ../localrepo/localrepo.py:200 ../localrepo/repo.py:80
msgid "Package is already in the repo: {0}"
msgstr "El paquet ja està en el repositori:{0}"

#: ../localrepo/localrepo.py:209 ../localrepo/localrepo.py:222
#: ../localrepo/localrepo.py:286
msgid "{0} packages found"
msgstr "{0} paquet trobat"

#: ../localrepo/localrepo.py:210
msgid "Starting an AUR upgrade"
msgstr ""

#: ../localrepo/localrepo.py:213
msgid "Nothing to do"
msgstr "Res a fer"

#: ../localrepo/localrepo.py:223
msgid "Checking for updates"
msgstr "Comprovant actualitzacions"

#: ../localrepo/localrepo.py:234
msgid "All packages are up to date"
msgstr "Tots els paquets estan al dia"

#: ../localrepo/localrepo.py:237 ../localrepo/localrepo.py:266
msgid "Upgrade?"
msgstr ""

#: ../localrepo/localrepo.py:246
msgid "Updating all VCS packages"
msgstr ""

#: ../localrepo/localrepo.py:247
msgid "Starting a VCS upgrade"
msgstr ""

#: ../localrepo/localrepo.py:251
msgid "No VCS packages found"
msgstr ""

#: ../localrepo/localrepo.py:261
msgid "No updates found"
msgstr ""

#: ../localrepo/localrepo.py:275
msgid "Clearing the cache"
msgstr ""

#: ../localrepo/localrepo.py:279
msgid "Cleared cache"
msgstr ""

#: ../localrepo/localrepo.py:287
msgid "Running integrity check"
msgstr "Executant comprovació d'integritat"

#: ../localrepo/localrepo.py:291
msgid "No errors found"
msgstr "No s'han trobat errors"

#: ../localrepo/localrepo.py:292
msgid "Finished integrity check without any errors"
msgstr ""

#: ../localrepo/localrepo.py:295
msgid "Finished integrity check with errors:"
msgstr ""

#: ../localrepo/localrepo.py:304
msgid "Restoring database"
msgstr "Restaurant la base de dades"

#: ../localrepo/localrepo.py:308
msgid "Restored Database"
msgstr ""

#: ../localrepo/log.py:52
msgid "Could not open log file: {0}"
msgstr ""

#: ../localrepo/log.py:65
msgid "[Error] {0}"
msgstr ""

#: ../localrepo/log.py:104
msgid "Could not store log file: {0} -> {1}"
msgstr ""

#: ../localrepo/log.py:140
msgid "Could not copy PKGBUILD dir: {0} -> {1}"
msgstr ""

#: ../localrepo/package.py:35
msgid "Unresolved dependencies: {0}"
msgstr ""

#: ../localrepo/package.py:101
msgid "Could not download file: {0}"
msgstr "No s'ha pogut descarregar l'arxiu:{0}"

#: ../localrepo/package.py:113
msgid "Could not open tarball: {0}"
msgstr ""

#: ../localrepo/package.py:120
msgid "Tarball contains bad member: {0}"
msgstr "Tarball conté un membre erroni:{0}"

#: ../localrepo/package.py:142
msgid "Could not extract tarball: {0}"
msgstr ""

#: ../localrepo/package.py:169 ../localrepo/repo.py:130
#: ../localrepo/repo.py:150 ../localrepo/repo.py:198
msgid "Could not list directory: {0}"
msgstr ""

#: ../localrepo/package.py:191
msgid "Could not find PKGBUILD: {0}"
msgstr ""

#: ../localrepo/package.py:211
msgid "Could not find any package: {0}"
msgstr ""

#: ../localrepo/package.py:237
msgid "File does not exist: {0}"
msgstr "Arxiu inexistent:{0}"

#: ../localrepo/package.py:245
msgid "Could not read package info: {0}"
msgstr ""

#: ../localrepo/package.py:253
msgid "An error occurred in tar"
msgstr "A succeït un error a tar"

#: ../localrepo/package.py:267
msgid "Could not determine package size: {0}"
msgstr ""

#: ../localrepo/package.py:269
msgid "Could not calculate package checksums: {0}"
msgstr ""

#: ../localrepo/package.py:288
msgid "Invalid file name: {0}"
msgstr "Nom d'arxiu invalid: {0}"

#: ../localrepo/package.py:373
msgid "Destination is no directory: {0}"
msgstr "El destí no és un directori:{0}"

#: ../localrepo/package.py:381
msgid "File already exists: {0}"
msgstr "L'arxiu ja existeix:{0}"

#: ../localrepo/package.py:387
msgid "Could not move package: {0} -> {1}"
msgstr ""

#: ../localrepo/package.py:398
msgid "Could not move sig file: {0} -> {1}"
msgstr ""

#: ../localrepo/package.py:409
msgid "Could not remove package: {0}"
msgstr ""

#: ../localrepo/pacman.py:20
msgid "An error occurred while running: {0}"
msgstr ""

#: ../localrepo/pacman.py:102
msgid "Could not change working directory: {0}"
msgstr ""

#: ../localrepo/pacman.py:143
msgid "Ooh no! Somebody killed the repo elephant"
msgstr "Ooh no! Algú ha matat la base de dades del elefant  "

#: ../localrepo/parser.py:55 ../localrepo/parser.py:62
msgid "Could not parse PKGBUILD: {0}"
msgstr ""

#: ../localrepo/parser.py:69
msgid "Missing PKGBUILD entry: {0}"
msgstr ""

#: ../localrepo/parser.py:98
msgid "Missing PKGINFO entry: {0}"
msgstr ""

#: ../localrepo/parser.py:123
msgid "Missing fields: {0}"
msgstr ""

#: ../localrepo/repo.py:91
msgid "Could not add packages to the db: {0}"
msgstr ""

#: ../localrepo/repo.py:108
msgid "Could not remove packages from the db: {0}"
msgstr ""

#: ../localrepo/repo.py:120
msgid "Package has no valid checksum: {0}"
msgstr "El paquet no presenta un suma d'integritat vàlida:{0}"

#: ../localrepo/repo.py:123
msgid "Missing signature for package: {0}"
msgstr ""

#: ../localrepo/repo.py:128
msgid "Package is not listed in repo database: {0}"
msgstr "El paquet no es troba llistat en la base de dades del repositori:{0}"

#: ../localrepo/repo.py:145
msgid "Could not find database: {0}"
msgstr ""

#: ../localrepo/repo.py:170
msgid "Could not open database: {0}"
msgstr ""

#: ../localrepo/repo.py:179
msgid "Invalid db entry: {0}: {1}"
msgstr ""

#: ../localrepo/repo.py:181
msgid "Could not read db entry: {0}"
msgstr ""

#: ../localrepo/repo.py:189
msgid "Could not close database: {0}"
msgstr ""

#: ../localrepo/repo.py:206
msgid "Could not remove database: {0}"
msgstr ""

#: ../localrepo/repo.py:217 ../localrepo/repo.py:219
msgid "Cache is outdated: {0}"
msgstr ""

#: ../localrepo/repo.py:225
msgid "Could not load cache: {0}"
msgstr ""

#: ../localrepo/repo.py:237
msgid "Could not update cache: {0}"
msgstr ""

#: ../localrepo/repo.py:245
msgid "Could not clear cache: {0}"
msgstr ""

#: ../localrepo/utils.py:96
msgid "[y|N]"
msgstr ""

#: ../localrepo/utils.py:97
msgid "y"
msgstr ""

#: ../localrepo/utils.py:97
msgid "yes"
msgstr ""

#: ../localrepo/utils.py:110
msgid "Architecture"
msgstr "Arquitectura"

#: ../localrepo/utils.py:111
msgid "Bugs"
msgstr ""

#: ../localrepo/utils.py:112
msgid "Build Date"
msgstr "Data de compilació"

#: ../localrepo/utils.py:113
msgid "Package size"
msgstr "Grandària del paquet"

#: ../localrepo/utils.py:114
msgid "Description"
msgstr "Descripció"

#: ../localrepo/utils.py:115
msgid "Filename"
msgstr "Nom Arxiu"

#: ../localrepo/utils.py:116
msgid "Installed size"
msgstr "tamany instal·lat"

#: ../localrepo/utils.py:117
msgid "Last update"
msgstr "Darrera actualització"

#: ../localrepo/utils.py:118
msgid "License"
msgstr "Llicència"

#: ../localrepo/utils.py:119
msgid "Location"
msgstr "Localització"

#: ../localrepo/utils.py:120
msgid "MD5sum"
msgstr "MD5sum"

#: ../localrepo/utils.py:121
msgid "Name"
msgstr "Nom"

#: ../localrepo/utils.py:122
msgid "Packager"
msgstr "Empaquetador"

#: ../localrepo/utils.py:123
msgid "Packages"
msgstr "Paquets"

#: ../localrepo/utils.py:124
msgid "Signed"
msgstr ""

#: ../localrepo/utils.py:125
msgid "SHA256sum"
msgstr "SHA256sum"

#: ../localrepo/utils.py:126
msgid "Translations"
msgstr ""

#: ../localrepo/utils.py:127
msgid "URL"
msgstr ""

#: ../localrepo/utils.py:128
msgid "Version"
msgstr "Versió"

#: ../localrepo/utils.py:129
msgid "Website"
msgstr ""

#: ../localrepo/utils.py:157
msgid "Yes"
msgstr ""

#: ../localrepo/utils.py:157
msgid "No"
msgstr ""

# Additional messages from the argparse module
# See: /usr/lib/python3.2/argparse.py
msgid "usage: "
msgstr ""

msgid "positional arguments"
msgstr ""

msgid "optional arguments"
msgstr ""

msgid "show this help message and exit"
msgstr ""

msgid "too few arguments"
msgstr ""

msgid "unrecognized arguments: %s"
msgstr ""

msgid "expected at least one argument"
msgstr ""

msgid "%(prog)s: error: %(message)s\n"
msgstr ""
